engine/swarm/services.md

```diff
@@ -70,6 +70,128 @@ $ docker service create --name helloworld alpine ping docker.com
 9uk4639qpg7npwf3fn2aasksr
 ```
 
+### Specify the image version the service should use
+
+When you create a service without specifying any details about the version of
+the image to use, the service uses the latest version available in Docker Hub
+or your registry. You can force the service to use a specific version of
+the image in a few different ways, depending on your desired outcome.
+
+An image version can be expressed in several different ways:
+
+- If you don't specify a version at all, by convention the image's `latest` tag
+  is resolved to a _digest_, which is a unique identifier for a version of the
+  image. The following two commands are equivalent:
+  ```bash
+  $ docker service create --name="myservice" ubuntu
+
+  $ docker service create --name="myservice" ubuntu:latest
+  ```
+
+- If you specify a different tag, Docker resolves that tag to a digest, and uses
+  that version.
+  ```bash
+  $ docker service create --name="myservice" ubuntu:16:04
+  ```
+
+- If you specify a digest directly, that exact version of the image is always
+  used.
+  ```bash
+  $ docker service create --name="myservice" ubuntu:16.04@sha256:35bc48a1ca97c3971611dc4662d08d131869daa692acb281c7e9e052924e38b1
+  ```
+
+**Note: Using `latest`** There is no guarantee that the version tagged
+with `latest` is actually the newest or most up-to-date version. In general, it
```

Comment:
The main reason to warn against `latest` is that over time, if image behind the `latest` tag updates, the service could end up with different versions. This text doesn't make that clear.